openslides/core/config_variables.py

```diff
@@ -155,3 +155,19 @@ def get_config_variables():
         label='Default countdown',
         weight=185,
         group='Projector')
+
+    # set the resolution for one projector. It can be removed with the multiprojector feature.
+    yield ConfigVariable(
+        name='projector_resolution',
+        default_value='1024_768',
+        input_type='choice',
+        label='Projector Resolution',
+        choices=(
+            {'value': '800_600', 'display_name': '800x600 (4:3)'},
+            {'value': '1024_768', 'display_name': '1024x768 (4:3)'},
+            {'value': '1280_1024', 'display_name': '1280x1024 (5:4)'},
+            {'value': '1360_768', 'display_name': '1360x768 (16:9)'},
+            {'value': '1680_1050', 'display_name': '1680x1050 (8:5)'},
+            {'value': '1920_1080', 'display_name': '1920x1080 (16:9)'}),
```

Comment:
How do you handle custom fields now? Like just adding a file somewhere and read and parse it in the config?